tests/unit_tests/bittensor_tests/utils/test_utils.py

```diff
@@ -530,5 +532,22 @@ class MockException(Exception):
             self.assertEqual(nonce_start_after_iteration, (initial_nonce_start + update_interval * TPB) % nonce_limit,  "nonce_start was not updated by the correct amount")
 
 
+class TestExplorerURL(unittest.TestCase):
+    network_map: Dict[str, str] = {
+        "nakamoto": "https://polkadot.js.org/apps/?rpc=wss://archivelb.nakamoto.opentensor.ai:9943/explorer",
+        "example": "https://polkadot.js.org/apps/?rpc=wss://example.example.com/explorer",
+        # "bad": None # no explorer for this network
```

Comment:
need to uncomment?

Reply:
it's a comment to show it should be none when not in the map